path-finding.asciidoc

```diff
@@ -221,15 +221,18 @@ If people stand in a line at the cash register for their groceries this would be
 Thus lightnign developers 
 
 
-==== Probing based improvements
-The last source of information that nodes could use is to probe the network themselves.
-Instead of making the actual payment nodes could send out many fake payments which are onions to a random payment hash.
-Given the properties of the hashfunction it is save to assume that noone would know the preimage.
-In that sense the payment will only fail at the destination and nodes can learn a lot about the balances.
-Of course this produces spam and heavy load on the network and it is not recommended that nodes do this.
-However participants cannot really be stopped from doing this.
-unless channel partners see a lot of traffic coming on a channel which always fails and never settles.
-In this case channel partners could decide to close the channel. 
+==== Improvements to probing
```

Comment:
not sure about the heading but as mentioned in the other PR the chapter will be rewritten. so I will address this